models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md

```diff
@@ -0,0 +1,84 @@
+# formula-recognition-polynomials-handwritten-0001 (composite)
+
+## Use Case and High-Level Description
+
+This is an im2latex composite model that recognizes latex formulas.
+The model uses vocabulary file to predict sequence of latex tokens.
+The model is built on the resnext50 backbone with additional attention-based text recognition head.
```

Comment:
would be nice to add what dataset model was trained on. If that is internal dataset then just say trained on internal dataset, containing handwritten symbols

Reply:
Thank you for your suggestion, the description was updated.